lib/registries/addon/overview/controller.ts

```diff
@@ -33,6 +35,14 @@ export default class Overview extends Controller {
 
     @alias('model.taskInstance.value') registration?: Registration;
 
+    @computed('router.currentRouteName')
+    get showMetadata() {
```

Comment:
I have a feeling we're going to need a `get isFilesPage()` computed, but I'm not opposed to having both, and we can add that in later.